bam_plcmd.c

```diff
@@ -949,6 +949,7 @@ static void print_usage(FILE *fp, const mplp_conf_t *mplp)
 "  -O, --output-BP         output base positions on reads\n"
 "  -s, --output-MQ         output mapping quality\n"
 "      --output-QNAME      output read names\n"
+"      --reverse-del       use '#' character to mark deletions on the reverse strand"
```

Comment:
`\n` and shorten to fit in 80 characters:

```suggestion
"      --reverse-del       use '#' character for deletions on the reverse strand\n"
```